tls/test_tls_cert.py

```diff
@@ -290,6 +296,25 @@ def test(self):
         self.check_cannot_start("with OID 1.2.840.10045.4.1 is unsupported")
 
 
+class InvalidKeyWithGoodCert(X509):
+    def setUp(self):
+        self.cgen = CertGenerator()
+        self.cgen.key = {"alg": "ecdsa", "curve": ec.SECP256R1()}
+        self.cgen.sign_alg = "sha256"
+        self.cgen.generate()
+
+        os.system("openssl ecparam -name secp192r1 -genkey -noout -out %s" % self.cgen.f_key)
+
+        self.tempesta = {
+            "config": X509.tempesta_tmpl % self.cgen.get_file_paths(),
+            "custom_cert": True,
+        }
+        tester.TempestaTest.setUp(self)
+
+    def test(self):
+        self.check_cannot_start_impl("with OID 1.2.840.10045.3.1.1 is unsupported")
```

Comment:
This test does not work with remote config. I suggest (it works for me):
```suggestion
        self.tempesta = {
            "config": X509.tempesta_tmpl % self.cgen.get_file_paths(),
            "custom_cert": True,
        }
        tester.TempestaTest.setUp(self)

    def test(self):
        cert_path, key_path = self.cgen.get_file_paths()
        remote.tempesta.copy_file(cert_path, self.cgen.serialize_cert().decode())
        remote.tempesta.copy_file(key_path, self.cgen.serialize_priv_key().decode())
        
        remote.tempesta.run_cmd("openssl ecparam -name secp192r1 -genkey -noout -out %s" % self.cgen.f_key)
        
        self.check_cannot_start_impl("with OID 1.2.840.10045.3.1.1 is unsupported")
```